tests/threadpool/test_wrap.py

```diff
@@ -1,10 +0,0 @@
-from aiofiles.threadpool import wrap
-
-import pytest
-
-
-def test_unsupported_wrap():
```

Comment:
Let's move this test somewhere instead of removing it?

Reply:
I have changed it, now it is better.

Reply:
BTW, could you open "Discussions" for this project?